src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -93,7 +93,9 @@ const parseLocalisedDateIfValid = (locale = 'en-UK', inputValue = '') => {
   );
 
   const isValid =
-    !Number.isNaN(dateAsUnixEpochInteger) && dateAsUnixEpochInteger >= 0;
+    // IE 11 doesn't support Number.isNaN (bleeding-edge ES6)
+    // eslint-disable-next-line no-restricted-properties
+    !window.isNaN(dateAsUnixEpochInteger) && dateAsUnixEpochInteger >= 0;
```

Comment:
please remove `window.` or something like this `(window || global).isNaN(..)` so that it could be used in node unit tests

Reply:
I removed `window`.